InputManipulation/CsvManipulator.cs

```diff
@@ -0,0 +1,232 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Text;
```

Comment:
zijn deze allemaal nodig?

Reply:
Text is voor stringbuilder wat veel sneller is dan string += string
IO is voor het lezen en het schrijven van de files
Globalization is voor de invariant culture, deze culture zorgt ervoor dat de getallen achter een punt staan ipv een comma
Diagnostics is misschien niet helemaal nodig maar het is een leuke toevoeging om te zien hoelang hij erover deed